fixtures/braid-design-system/src/App.test.tsx

```diff
@@ -15,8 +15,8 @@ describe('braid-design-system', () => {
     ).toMatchInlineSnapshot(`
       "<style type="text/css">
                   html,body{margin:0;padding:0;background:#F7F8FB}
-                  html.sprinkles_darkMode__in90cbz,html.sprinkles_darkMode__in90cbz body{color-scheme:dark;background:#0E131B}
-                </style><div class="apacTheme_default__eah7240 typography_lightModeTone_light__myxsed18 typography_darkModeTone_dark__myxsed1b"><div class="reset_base__pv0jxi0 sprinkles_paddingTop_large_mobile__in90cb6q App_vanillaBox__inn18b0"></div></div>"
+                  html.sprinkles_darkMode__1xl4lgvz,html.sprinkles_darkMode__1xl4lgvz body{color-scheme:dark;background:#0E131B}
```

Comment:
Weird, why did these hashes change?

Reply:
Maybe this? https://github.com/seek-oss/sku/pull/772/files#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR6406
Honestly not too sure what causes the hash to change half the time.

Reply:
There's a Braid above and another one below the one you linked 

![braids galore](https://user-images.githubusercontent.com/3297808/232406761-d57f728d-caff-474b-85dd-a55da6cd3afa.png)

Reply:
> Honestly not too sure what causes the hash to change half the time.

Vanilla Extract computes the hash from the file scope. The file scope contains the file path and when the location of `braid-design-system` changes, so does the hash.